I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang-16.txt

```diff
@@ -0,0 +1,8232 @@
+   Node-2.  This is illustrated in Figure 30.
+
+   In this use case, the connectivity is reflected in the connectivity-
+   matrix describing the connectivity between the LTPs representing an
+   add-drop port in WDM-TE-Node-2 connected to the transceiver of a
+   remote OT and the LTPs associated with the different ROADM degrees
+   including the optical impairments for the different paths.
+
+     WDM-TE-node 1                   WDM-TE-node 2
+
+                  +--------------------------------------------------+
+                  |  LTP 20                               ___        |
+      +-------+   |  +-------+     AD 1  ___             |   \  Line |
+      |   TTP1|   |  |    ---o-----     /   |       /----o    | LTP 1|
+      |  +----|   |  |   /   |     \   |    o------/     | 1  o------o->
+    --o->| Tx o---o->o--o  5 |      \--o    |           -o    |      |
+      |  +----|   |  |   \   |         |  4 |          / |___/       |
+    <-o--| Rx o   |  |    ---o-\    /--o    |         /         DEG 1|
+      |  +----|   |  +-------+  \   |  |    o-       /    ___        |
+      |       |   |   Splitter   \  |   \___| \     /    |   \  Line |
+      |       |   |  +-------+   |  |          \         o    | LTP 2|
+      |   TTP2|   |  |    ---o---|--/           \ /      | 2  o------o->
+      |  +----|   |  |   /   |   |       ___     \   /---o    |      |
+    --o->| Tx o---o->o--o  5 |    \     /   |   / \ /    |___/       |
+      |  +----|   |  |   \   |     \   |    o--/   \           DEG 2 |
+    <-o--| Rx o   |  |    ---o--\   \--o    |     / \     ___        |
+      |  +----|   |  +-------+   \     |    |    /   \   |   \  Line |
+      |       |   |  LTP 30       \----o  6 |   /     \--o    | LTP 3|
+      |   TTP3|   |                    |    o--/         | 3  o------o->
+      |  +----|   |          /-------->o    |            o    |      |
+    --o->| Tx o---o---------/          |    |            |___/       |
+      |  +----|   |  LTP 40             \___|                  DEG 3 |
+    <-o--| Rx o   |                AD 2                              |
+      |  +----|   |                                                  |
+      |       |   |                                                  |
+      +-------+   |                                                  |
+                  |                                                  |
+                  +--------------------------------------------------+
+
+
+
+
+Beller, et al.           Expires 9 January 2025               [Page 111]
+
+Internet-Draft    Opt. Impairment-Aware Topo YANG Model        July 2024
+
+
+     Figure 30: JSON Example for Restricted Connectivity between ROADM
+                   Add-Drop Ports and ROADM Degree Ports
+
+   {
+     "roadm-path-impairments": [
+       {
+         "roadm-path-impairments-id": 1,
+         "roadm-add-path":
+         "Add path impairments from LTP 10 to LTP 1 or LTP 2 or LTP 3."
```

Comment:
There is no LTP10 on figure 30

Reply:
@EstherLerouzic  LTP 10 is not considered in the example since in the ascii figure is reported restricted connectivity only. We will delete the LTP 10 part.